i18n/en.yaml

```diff
@@ -9,7 +9,7 @@ hero: Receive and send emails anonymously
+  {"@context":"https://schema.org/","@type":"Product","name":"Email Alias","image":"https://simplelogin.io/images/hero.svg","description":"With email aliases, you can be anonymous online and protect your inbox against spams and phishing. Open source. Made and hosted in Switzerland.","brand":{"@type":"Brand","name":"SimpleLogin"},"aggregateRating":{"@type":"AggregateRating","ratingValue":"4.9","bestRating":"5","worstRating":"0","ratingCount":"6","reviewCount":"6"},"review":[{"@type":"Review","reviewBody":"Once I used it, it was a game changer. SimpleLogin is in my book criminally underrated.","reviewRating":{"@type":"Rating","ratingValue":"5","bestRating":"5","worstRating":"0"},"author":{"@type":"Person","name":"Techlore"}},{"@type":"Review","reviewBody":"SimpleLogin seems to me ahead of the game. It's all open-source and I mean everything, the apps, the service, the website.","reviewRating":{"@type":"Rating","ratingValue":"5","bestRating":"5","worstRating":"0"},"author":{"@type":"Person","name":"Michael Bazzell"}},{"@type":"Review","reviewBody":"I like to use SimpleLogin for trowaway accounts like restaurant reservations or newsletters. I opt for SimpleLogin because it allows me easily to shut off an address. SimpleLogin is a very effective SPAM control measure.","reviewRating":{"@type":"Rating","ratingValue":"5","bestRating":"5","worstRating":"0"},"author":{"@type":"Person","name":"Naomi Brockwell"}},{"@type":"Review","reviewBody":"I personaly use SimpleLogin as my email aliasing tool. SimpleLogin login is an important part of my strategy.","reviewRating":{"@type":"Rating","ratingValue":"5","bestRating":"5","worstRating":"0"},"author":{"@type":"Person","name":"Josh, All Things Secured"}},{"@type":"Review","reviewBody":"I usually have to enter my email while subscribing to new services online, albeit without much confidence. SimpleLogin gives me a trustworthy alternative and helps me controlling my privacy on the Internet.","reviewRating":{"@type":"Rating","ratingValue":"5","bestRating":"5","worstRating":"0"},"author":{"@type":"Person","name":"Nguyen Nhat Cuong"}},{"@type":"Review","reviewBody":"SimpleLogin gives me the convenience of a social login button with the privacy of an anonymous email address. I like that some of the money I spend on the service goes toward paying website owners, it allows me to feel less guilty about using uBlock Origin.","reviewRating":{"@type":"Rating","ratingValue":"5","bestRating":"5","worstRating":"0"},"author":{"@type":"Person","name":"Max Schumacher"}}]}
```

Comment:
the "hosted in Switzerland" isn't actually accurate as our data centers can be in other EU countries like Germany

Reply:
@nguyenkims fixed! :)